sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -393,8 +393,8 @@ def verify(self, algorithm, digest, signature, **kwargs):
 
         :param algorithm: verification algorithm
         :type algorithm: :class:`~azure.keyvault.keys.crypto.SignatureAlgorithm`
-        :param bytes digest:
-        :param bytes signature:
+        :param bytes digest: the signed bytes to verify
+        :param bytes signature: the signature we are verifying
```

Comment:
The digest is a hash of the plaintext message (i.e. it isn't signed):
```suggestion
        :param bytes digest: Pre-hashed digest corresponding to **signature**. The hash algorithm used must be
          compatible with **algorithm**.
        :param bytes signature: signature to verify
```